src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -1157,6 +1157,21 @@ void M2(string[] args)
 //]
     }}
 }}
+";
+
+        private static readonly string s_preferUTF8StringLiterals = $@"
+using System;
+
+//[
+class WebResponse
+{{
+    // {ServicesVSResources.Prefer_colon}
+    private readonly byte[] header = ""Hello""u8;
+
+    // {ServicesVSResources.Over_colon}
+    private readonly byte[] header = new byte[] {{ 72, 101, 108, 108, 111 }};
```

Comment:
do we have a test of `private readonly byte[] header = new[] {{ 72, 101, 108, 108, 111 }};` ?  (implicit array creation?)

Reply:
There is a test for `new[] { (byte)65, (byte)66, (byte)67 }`, yes. What you have there doesn't actually compile 😛

Reply:
Sorry. I meant a 2d array of bytes :-)